### PR TITLE
Remove name of user from email template

### PR DESCRIPTION
**Pull request by marcoconti83**
User names and team names have been removed from all emails that are generated as result of hitting non-authenticated endpoints.

This will make the invitation email contain no user input other than the email address itself, reducing the risk of spam.

The commit also includes some translations: UK-DE-SV-IT-FR.

### CODE REVIEW

File: services/brig/deb/opt/brig/templates/user/de/email/team-activation.html [hunk 1]
```diff
@@ -79,18 +81,17 @@
               <![endif]-->
           </p>
           <p style="font-size: 10pt; font-weight: 300; line-height: 1.5em; color: #000;">
-            Bitte klicke open auf "Verifizieren", um deine Identität zu bestätigen. Erst danach kannst du Wire nutzen.
+            Bitte klicke oben auf "Verifizieren", um deine Identität zu bestätigen. Danach kannst du Wire nutzen.
           </p>
           <p style="font-size: 10pt; font-weight: 300; line-height: 1.5em; color: #000;">
-              Wenn du nicht auf den "Verifizieren"-Button klicken kannst, benutze stattdessen bitte folgenden Link: <br>
+            Wenn du nicht auf den "Passwort ändern"-Button klicken kannst, benutze stattdessen bitte folgenden Link:<br>
```
Comment (tiago-loureiro) [inline services/brig/deb/opt/brig/templates/user/de/email/team-activation.html]: This is for team activation

File: services/brig/deb/opt/brig/templates/user/de/email/team-activation.html [hunk 2]
```diff
@@ -79,18 +81,17 @@
...
           </p>
           <p style="font-size: 10pt; font-weight: 300; line-height: 1.5em; color: #BAC8D1; overflow-wrap: break-word; word-wrap: break-word; -ms-word-break: break-all; word-break: break-word;">
             ${url}
           </p>
           <p style="font-size: 10pt; font-weight: 300; line-height: 1.5em; color: #000;">
-              Wenn du kein Wire-Benutzerkonto mit dieser E-Mail-Adresse erstellt hast, klicke bitte <a href="https://support.wire.com/hc/de/requests/new" style="color: #3898DD; font-weight: 400; text-decoration:none!important;">hier</a>.
+            Wenn du kein Wire-Benutzerkonto mit dieser E-Mail-Adresse erstellt hast, klicke bitte <a href="http://support.wire.com/hc/en-us/requests/new" style="color: #3898DD; font-weight: 400; text-decoration:none!important;">hier</a>
```
Comment (tiago-loureiro) [inline services/brig/deb/opt/brig/templates/user/de/email/team-activation.html]: Links to the english support; also, uses `http`

File: services/brig/deb/opt/brig/templates/user/de/email/activation.html [hunk 3]
```diff
@@ -48,7 +48,7 @@
 
           <p style="
               color: #000000;">
-            deine E-Mail-Adresse <strong>${email}</strong> wurde benutzt, um ein Wire-Benutzerkonto zu erstellen. Wir möchten gerne sicherstellen, dass du <strong>${name}</strong> bist.
+            <strong>${email}</strong> wurde verwendet, um ein Draht-Konto erstellen. Überprüfe deine e-Mail-Adresse.
```
Comment (tiago-loureiro) [inline services/brig/deb/opt/brig/templates/user/de/email/activation.html]: `Draht-Konto`? And _zu_ erstellen.

File: services/brig/deb/opt/brig/templates/user/de/email/activation.html [hunk 4]
```diff
@@ -77,23 +77,24 @@
             display: inline-block;
             margin: 20px 0;
             line-height: 3;
-                ">VERIFIZIEREN</a>
+            text-align: center;
+                ">Verifizieren</a>
```
Comment (tiago-loureiro) [inline services/brig/deb/opt/brig/templates/user/de/email/activation.html]: Everywhere else we use uppercase, can we keep it here too?

File: services/brig/deb/opt/brig/templates/user/de/email/activation.html (hunk 3 above)
Comment (jschaul) [reply services/brig/deb/opt/brig/templates/user/de/email/activation.html]: `<strong>${email}</strong> wurde verwendet, um ein Wire Benuzterkonto zu erstellen.` (The `Überprüfe` bit is unnecessary as the next sentence already states the same.

File: services/brig/deb/opt/brig/templates/user/en/email/activation.html [hunk 5]
```diff
@@ -48,7 +48,7 @@
 
           <p style="
               color: #000000;">
-            <strong>${email}</strong> was used to create a Wire account. We want to verify that you are indeed <strong>${name}</strong>.
+            <strong>${email}</strong> was used to create a Wire account. Please, verify your email.
```
Comment (jschaul) [inline services/brig/deb/opt/brig/templates/user/en/email/activation.html]: Is it really necessary to emphasize the 'please' part by introducing a comma here? To me, no comma reads more natural in this context (see https://english.stackexchange.com/questions/389/in-what-cases-should-i-use-a-comma-after-please)

File: services/brig/deb/opt/brig/templates/user/de/email/team-activation.html [hunk 6]
```diff
@@ -41,10 +41,12 @@
 <tr>
     <td width="500" colspan="2" style="padding-top: 48px;padding-bottom: 0px;">
     <p style="font-size: 32px; font-weight: 300; text-align: center; line-height: 1.3;">
-    Dein neues Wire-Benutzerkonto im '${team}' Team
+    Dein neues Wire Konto
```
Comment (jschaul) [inline services/brig/deb/opt/brig/templates/user/de/email/team-activation.html]: We use `Wire-Benutzerkonto` everywhere else, please ensure the use is consistent across all emails (either `Wire Konto` or `Wire-Benutzerkonto`. )

File: services/brig/deb/opt/brig/templates/user/de/email/team-activation.html [hunk 7]
```diff
@@ -102,7 +103,7 @@
                   color: #B9BCBD;
                   font-weight: 500;
                   text-decoration:none !important;
-                ">Datenschutzrichtlinie</a><span style="margin-left: 8px; margin-right: 8px; color: #B9BCBD;">&middot;</span>
+                ">Datenschutz</a><span style="margin-left: 8px; margin-right: 8px; color: #B9BCBD;">&middot;</span>
```
Comment (jschaul) [inline services/brig/deb/opt/brig/templates/user/de/email/team-activation.html]: In other changes in this PR, `Datenschutz` was changed to `Datenschutzrichtlinie`. Here, the reverse is done. `Datenschutzrichtlinie` is preferred. Please ensure consistency.

File: services/brig/deb/opt/brig/templates/user/de/email/verification.html [hunk 8]
```diff
@@ -1 +0,0 @@
-../../en/email/verification.html
```
Comment (jschaul) [inline services/brig/deb/opt/brig/templates/user/de/email/verification.html]: `Gib diesen Code ein, um deine e-Mail-Adresse zu überprüfen und erstelle dein Konto.` should be changed to `Gib diesen Code ein, um deine e-Mail-Adresse zu überprüfen und dein Konto zu erstellen.`

File: services/brig/deb/opt/brig/templates/user/uk/email/verification.txt [hunk 9]
```diff
@@ -1 +0,0 @@
-../../en/email/verification.txt
```
Comment (jschaul) [inline services/brig/deb/opt/brig/templates/user/uk/email/verification.txt]: Line 6 is clearly not translated.

File: services/brig/deb/opt/brig/templates/user/de/email/team-activation.html (hunk 6 above)
Comment (marcoconti83) [reply services/brig/deb/opt/brig/templates/user/de/email/team-activation.html]: Thanks for the suggestions. At this point in time, I would like to fix only broken links or major grammar errors. Our copywriter will review the copy of all emails this week to ensure consistency, so expect a better copy/translation to come in this or next week.

**Comment by marcoconti83** [comment]
I updated the translations. Please have another look.

**Pull request merged**